sys/auto_init/netif/auto_init_netdev_tap.c

```diff
@@ -8,7 +8,7 @@
  */
 
 /**
- * @ingroup auto_init_gnrc_netif
+ * @ingroup auto_init_netif
```

Comment:
Is there a reason why the `@ingroup` is only changed here?

Reply:
Not sure...both groups don't exist... will revert this change and we figure out something else in a separate PR.